resteasy-wadl/src/main/java/org/jboss/resteasy/wadl/ResteasyWadlWriter.java

```diff
@@ -395,7 +395,9 @@ public Result createOutput(final String namespaceUri, final String suggestedFile
                    Iterator<Include> iter = grammars.getInclude().iterator();
                    while (iter.hasNext()) {
                        for (Doc doc : iter.next().getDoc()) {
-                           if ("Generated".equals(doc.getTitle())) ;
+                           // CHECKSTYLE.OFF: EmptyStatement
+                           if ("Generated".equals(doc.getTitle()));
```

Comment:
Checkstyle reports "empty statement" here. I wasn't sure what to do with it.
I mentioned it here: https://github.com/resteasy/Resteasy/pull/1649#discussion_r229681375